### PR TITLE
support remote access.

### LINKED ISSUE #9: remote access support
https://github.com/alexmojaki/heartrate/blob/7fa29d9d5b2f72fdab98a1349efa5dfc784ab78a/heartrate/core.py#L86

### PR DESCRIPTION
**Pull request by lvelvee**
close #9

### CODE REVIEW

File: heartrate/core.py [hunk 1]
```diff
@@ -83,7 +83,8 @@ def include_file(path):
     totals = defaultdict(Counter)
 
     app = Flask(__name__)
-    app.config["SERVER_NAME"] = "localhost:{port}".format(port=port)
+    if host_is_local:
+        app.config["SERVER_NAME"] = "{host}:{port}".format(host=host, port=port)
```
Comment (alexmojaki) [inline heartrate/core.py:89]: Are you sure changing the default value from `localhost:{port}` to `127.0.0.1:{port}` is right?

File: heartrate/core.py [hunk 2]
```diff
@@ -70,7 +70,7 @@ def trace(
 ):
     calling_frame = inspect.currentframe().f_back
     calling_file = calling_frame.f_code.co_filename
-
+    host_is_local = host in ["127.0.0.1", "localhost"]
```
Comment (alexmojaki) [inline heartrate/core.py]: Please move this to where it's first used.

File: heartrate/core.py (hunk 1 above)
Comment (lvelvee) [reply heartrate/core.py:89]: Test case passed.

heartrate.trace(port=9999,host='localhost',browser=True,daemon=False, )
heartrate.trace(port=9999,host='127.0.0.1',browser=True,daemon=False, )

**Comment by alexmojaki** [comment]
Hi, sorry I don't know what happened here, I completely forgot/missed this and just stumbled into it again now. Looks good, thank you!

**Pull request merged**